### ISSUE TITLE
Lint for spacing after comment

### ISSUE THREAD

**Issue by pvichivanives** [rust-lang/rust-clippy#13709]
### What it does

A lint that would automatically add a space after a comment, common to many conventions and used in most example code.

### Advantage

Helps code-writers not accidentally quickly type up comments and forget to add a space after their comment. This would be a nice style lint that would keep internal code documentation more uniform. 

### Drawbacks

Maybe should be allow by default as there are some other conventions that do not enforce this. 

Could take more time to compile

### Example

```rust
//Hello World
```

Could be written as:

```rust
// Hello World
```

**Comment by y21** [comment]
For what it's worth, rustfmt already covers this and correctly formats it with `normalize_comments = true`

**Comment by pvichivanives** [comment]
Thanks! Will follow on that then!

**Comment by your-diary** [comment]
> For what it's worth, rustfmt already covers this and correctly formats it with normalize_comments = true

Is `normalize_comments` really relevant?

[The official doc](https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#normalize_comments) says

> `normalize_comments`
> 
> Convert `/* */` comments to `//` comments where possible

---

> A lint that would automatically add a space after a comment, common to many conventions and used in most example code.

[The Rust Style Guide](https://doc.rust-lang.org/style-guide/#comments) also says

> When using line comments, put a single space after the opening sigil.
